src/components/style/Filters.css

```diff
@@ -0,0 +1,33 @@
+
+.filters-holder img {
+    margin: 1em;
+}
+
+#form {
+    margin: 1em;
+    width: max-content;
+    height: max-content;
+}
+
+#form * {
+    margin: 1em;
+}
+
+#img-description {
+    max-width: 80%;
+    display: inline-block;
+}
+
+@media (max-width: 1435px) {
+    .filters-holder {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+}
+
+@media(max-width: 600px) {
+    .filters-holder img {
+        width: 80%;
+    }
+}
```

Comment:
Try using a strange viewport width like 1442px and you'll see that the effect you wanted to get rid of is still present. Some things that might be worth experimenting with:
* Grouping together the latter two items so that when your flex container wraps the elements, it keeps them together.
* Not using flexbox at all, and instead controlling the display style of your elements through a single `@media` query wholesale.